contracts/factories/TokenProxyFactory.sol

```diff
@@ -85,7 +85,7 @@ contract TokenProxyFactory is Factory {
         uint256 _requirements,
         string memory _name,
         string memory _symbol
-    ) public pure returns (bytes32) {
+    ) private pure returns (bytes32) {
```

Comment:
why? Isn't that function needed by the webapp as well?

Reply:
No, it is not. If the web app wants to know an address without revealing the details yet, it can calculate salt off-chain and call `predictProxyAddress(bytes32 _salt)`. Otherwise, it can call 
```
predictProxyAddress(
        bytes32 _rawSalt,
        address _trustedForwarder,
        IFeeSettingsV2 _feeSettings,
        address _admin,
        AllowList _allowList,
        uint256 _requirements,
        string memory _name,
        string memory _symbol
    )
```

Using `getSalt(...)` and then `predictProxyAddress(bytes32 _salt)` is just more complicated.